src/Context.ts

```diff
@@ -586,6 +587,14 @@ export class Context {
   rect(x: number, y: number, width: number, height: number) {
     this._context.rect(x, y, width, height);
   }
+  /**
+   * roundRect function.
+   * @method
+   * @name Konva.Context#roundRect
+   */
+  roundRect(x: number, y: number, width: number, height: number, radii: number) {
```

Comment:
Should type of `radii` by also an array? Like `number | Array<number>`?

Reply:
Oh you're right, missed the fact that you can fact an array of corners name 👍

Reply:
It won't be compatible with [the internal `drawRoundedRectPath`](https://github.com/konvajs/konva/pull/1695#issuecomment-1871972576) function
So let's just use the native API then 👍